src/test/java/org/zeromq/TestRapidOpenCloseSocket.java

```diff
@@ -0,0 +1,75 @@
+package org.zeromq;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import org.zeromq.ZMQ.Socket;
+
+// For issue: https://github.com/zeromq/jeromq/issues/198
+public class TestRapidOpenCloseSocket
+{
+  @Test
+  public void testRapidOpenCloseSocket() throws Exception
```

Comment:
Please add @Ignore, so the failing test is not blocking CI. Adding timeout on test could also be useful.